continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -27,6 +27,8 @@ requirements:
     - python >=3.8
     - pip
     - dask-core {{ dask_version }}
+    - versioneer =0.28
+    - tomli # [py<311]
```

Comment:
If we build on 3.11 tomli isn't needed, my understanding is noarch builds still build a package for each python version?

Reply:
While true, [versioneer has `try...except...` logic]( https://github.com/python-versioneer/python-versioneer/blob/1299bd1e6334889ea90aa254af0c3825d62344d5/src/header.py#L25-L32 ) to prefer the builtin when available. So it probably doesn't matter much whether the selector is here or not

Think Charles' point is we generally don't want to use selectors in a `noarch` file since it can't really be conditioned. That said, with `build` & `host` dependencies this is less relevant. Since those are just used for the build and the packages wind up with a fixed set of dependencies regardless

This all to say there are valid reasons to drop this or keep it. However in terms of the end result, it shouldn't matter which way we go

Reply:
Thanks for the clarification @jakirkham - I'm fine going either way here, just wanted to verify that this had no impact on the build